desktop-app/app/components/ScrollControls/index.js

```diff
@@ -60,6 +62,13 @@ const ScrollControls = ({
             </div>
           </Tooltip>
         </Grid>
+        <Grid item className={cx(commonStyles.icons, commonStyles.enabled)}>
+          <Tooltip title="Disable event mirroring">
+            <div onClick={toggleEventMirroringAllDevices}>
```

Comment:
@ns23 I think you will have to activate the button when the mirroring is disabled so that the users are aware of the state.

Reply:
Got it ..I will look into it.

Reply:
@ns23 please address this request